src/main/java/calculator/CalculatorController.java

```diff
@@ -0,0 +1,45 @@
+package calculator;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CalculatorController {
+        private final CalculatorServiсe calculatorServiсe;
+
+    public CalculatorController(CalculatorServiсe calculatorServiсe) {
+        this.calculatorServiсe = calculatorServiсe;
+    }
+
+    @GetMapping(path = "/calculator")
+    public String Calculator() {
+        return "<b><i>Добро пожаловать в калькулятор</i></b>";
+    }
+
+    @GetMapping(path = "/calculator/plus")
+    public String CalculatorPlus(@RequestParam("num1") Integer num1, @RequestParam("num2") Integer num2) {
```

Comment:
если имена параметров совпадают, то можно упростить
```suggestion
    public String CalculatorPlus(@RequestParam Integer num1, @RequestParam Integer num2) {
```